static/app/views/settings/project/server-side-sampling/utils/projectStatsToPredictedSeries.tsx

```diff
@@ -28,38 +28,54 @@ export function projectStatsToPredictedSeries(
     stack: 'predictedUsage',
   };
 
-  const seriesData: Record<string, Series['data']> = {
-    accepted: [],
-    droppedServer: [],
-    droppedClient: [],
+  const seriesData: Record<
+    'indexedAndProcessed' | 'processed' | 'discarded',
+    Series['data']
+  > = {
+    indexedAndProcessed: [],
+    processed: [],
+    discarded: [],
   };
 
   (
     projectStats.intervals.map((interval, index) => {
-      const result = {};
+      const result = {
+        indexedAndProcessed: 0,
+        processed: 0,
+        discarded: 0,
+      };
       projectStats.groups.forEach(group => {
-        result[group.by.outcome] = group.series[quantityField][index];
+        switch (group.by.outcome) {
+          case Outcome.ACCEPTED:
+            result.indexedAndProcessed += group.series[quantityField][index];
+            break;
+          case Outcome.CLIENT_DISCARD:
+            result.discarded += group.series[quantityField][index];
+            break;
+          case Outcome.FILTERED:
+            if (String(group.by.reason).startsWith('Sampled')) {
```

Comment:
is it more secure to use uppper/lower so we don't fail if we get:
`SAMPLED` or `sampled`?

```
> let a = "SAMPLED";
undefined
> a.startsWith('Sampled');
false
> a.toLowerCase().startsWith('sampled');
true
```

Reply:
We should not get SAMPLED or sampled. The value comes from the ingest team so I would like to keep it as explicit as possible. This is also temporary as we'll get a proper outcome for this towards the GA.